### ISSUE TITLE
Translation with AI

### ISSUE THREAD

**Issue by brishtibheja** [ankitects/anki#5469]
The manual's [translated versions](https://docs.ankiweb.net) we currently list are not all actively maintained. Many are incomplete, outdated or is accessible only from web archive's extremely slow website.

Following the discussion [here](https://forums.ankiweb.net/t/japanese-translation-of-ankimanual-anki-faqs-ankimobile-manual-and-fsrs-wiki/48382?u=sorata), some of the translations can be redone using AI (and preferably served using GitHub pages/mdBook). For consistency in terminology, I have suggested using a glossary in the prompt.

**Comment by Rafaelorr** [comment]
I'm a native Dutch speaker, so I could check the Dutch translation of the manual.

**Comment by Rafaelorr** [comment]
> I'm a native Dutch speaker, so I could check the Dutch translation of the manual.

But I new to contributing to other people's repos, so I don't really know how to begin with your repo.

**Comment by GithubAnon0000** [comment]
Instructions aren't verly clear right now IMO. Before the merging of both repos, it was more straightforward. But the anki team will likely iron it out (e.g. I'd expect a mention at [contributing.md](https://github.com/ankitects/anki/blob/main/docs/contributing.md#translations)).

Anyways: if I'm not mistaken, new translations are done in `docs-site/`. You can check out the [README](https://github.com/ankitects/anki/blob/main/docs-site/README.md).

**Comment by abdnh** [comment]
Created an issue to track that: #5510

**Comment by Rafaelorr** [comment]
Hey, just a question. Is it normal that there is so little stuff translated for the non-english languages? I looked in the french version of the docs (my french is really bad, so don't ask me for the french docs), just to make sure and it just was a link to the internet archive.

**Comment by abdnh** [comment]
That's normal. Translations are not required to be complete to be included, and the French translation specifically is at least 6 years old at this point.

**Comment by Rafaelorr** [comment]
So I looked at the docs linked in the readme. I want to know if my understanding is correct. So I have to make a separate folder for the dutch translation (I'm calling it nl, because that's the dutch abveriation for dutch), copy the english files in that folders while maintaining the structure, translate them and  link to the new files in the docs.json under a new language ? I know that I'm asking a lot of questions, but I just want to be sure before wasting hours on the wrong approaches.

**Comment by abdnh** [comment]
That's correct. You can see an example with the Arabic translation in this PR: https://github.com/ankitects/anki/pull/5483

Feel free to ask questions!

**Comment by Rafaelorr** [comment]
This is my fork of the anki docs. https://github.com/Rafaelorr/anki . So if another dutch speaker wants to help that would be appriacted. 
I'm planning on gradually committing every translated page when I'm done with it and git quashing every major section together. 
I'm starting with just the manual for the desktop version. 
I had some trouble adding dutch to languages section of docs.json, so I would like some help with that if when I'm done with the manual. I'm also a student, so my progress can be a bit slow.

**Comment by abdnh** [comment]
Once #5483 is merged and you have at least one page translated, you can submit a PR and we can help with any issues. No need to wait until completing the translation.

**Comment by Rafaelorr** [comment]
Ok, good to know. I'm just doing the files in the manual folder. Just 55 files, can't be to big.

**Comment by abdnh** [comment]
You can also post on the [forums](https://forums.ankiweb.net/) to see if there are any users interested in helping with translation/proofreading.

**Comment by Rafaelorr** [comment]
This is my post. https://forums.ankiweb.net/t/starting-dutch-translation/70934
